sdcm/utils/xml_utils.py

```diff
@@ -0,0 +1,54 @@
+import re
```

Comment:
why are you rebuilding the repoParser module ? how it's related to what this task is about ?